### PR TITLE
v2rayA.v2rayA version 2.0.1-1

### PR DESCRIPTION
**Pull request by MarksonHon**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/92325)

### CODE REVIEW

**Comment by MarksonHon** [comment]
There is an error that I cannot use winget to install version 2.0.1, so I submit it again as version 2.0.1-1.

File: manifests/v/v2rayA/v2rayA/2.0.1-1/v2rayA.v2rayA.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: v2rayA.v2rayA
+PackageVersion: 2.0.1-1
```
Comment (Trenly) [inline manifests/v/v2rayA/v2rayA/2.0.1-1/v2rayA.v2rayA.installer.yaml:5]:
```suggestion
PackageVersion: 2.0.1
```
Must match control panel due to the already existing versions

File: manifests/v/v2rayA/v2rayA/2.0.1-1/v2rayA.v2rayA.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: v2rayA.v2rayA
+PackageVersion: 2.0.1-1
```
Comment (Trenly) [inline manifests/v/v2rayA/v2rayA/2.0.1-1/v2rayA.v2rayA.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.0.1
```
Must match control panel due to the already existing versions

File: manifests/v/v2rayA/v2rayA/2.0.1-1/v2rayA.v2rayA.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: v2rayA.v2rayA
+PackageVersion: 2.0.1-1
```
Comment (Trenly) [inline manifests/v/v2rayA/v2rayA/2.0.1-1/v2rayA.v2rayA.yaml:5]:
```suggestion
PackageVersion: 2.0.1
```
Must match control panel due to the already existing versions

**Review by Trenly — CHANGES_REQUESTED** [review]
Please change the existing 2.0.1 manifest to use the new URLs, as the version hasn't changed in control panel, so there is no way for winget to differentiate 2.0.1 and 2.0.1-1

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
You would also need to rename the version directory under `manifests/v/v2rayA/v2rayA` from `2.0.1-1` to `2.0.1`.

One quick way to do it would be right here in GitHub Web by pressing the <kbd>.</kbd> (dot) key that'll open this PR in VSCode Online. Find and rename the directory from explorer tab on the left. Afterwards, you can stage and commit your changes from the Source Control tab on the left (the changes are automatically pushed)

**Pull request closed without merging**